src/WebJobs.Script.WebHost/WebHooks/WebHookReceiverManager.cs

```diff
@@ -68,9 +68,9 @@ public WebHookReceiverManager(SecretManager secretManager)
             // so our custom WebHookHandler can invoke it at the right time
             request.Properties.Add(AzureFunctionsCallbackKey, invokeFunction);
 
-            // TODO: Is there a better way? Requests content can't be read multiple
+            // Request content can't be read multiple
             // times, so this forces it to buffer
-            await request.Content.ReadAsStringAsync();
+            await request.Content.LoadIntoBufferAsync();
```

Comment:
Doesn't really change much, just a little more correct. The previous line also forced it to buffer, but we don't want to make any content type assumptions here.